js/script.js

```diff
@@ -89,5 +156,5 @@ const showError = (message) => {
 };
 
 $btnCreate.addEventListener("click", openForm);
-document.getElementById("btnAddTask").addEventListener("click", addTask);
+document.getElementById("btnAddTask").addEventListener("click", saveTask);
```

Comment:
Why are we using the same button for both add and update?

Reply:
Now in the modified code we are using only one button for addTask, and the click event for saving the task is attached with its respective saveButton. And it's inside the newly created function createEditElements which will be triggered while in the edit mode only.